.husky/commit-msg

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+. "$(dirname "$0")/_/husky.sh"
+
+npx --no-install commitlint --edit 
```

Comment:
I'd actually  never seen "no-install" before

https://docs.npmjs.com/cli/v7/commands/npx#compatibility-with-older-npx-versions 
> The --no-install option is deprecated, and will be converted to --no.

Reply:
I remember coming across something about it being deprecated. But there were a few conflicting sources I found so I just left it as is. But yeah, I can go ahead and change it.

Reply:
I pushed up the new change. I did however encounter a weird error that I had to bypass once I changed the Husky commit-msg script from `--no-install` to `--no`.   

The error I was getting was this: 
>→ No staged files match any configured task.
> @commitlint/cli@16.2.1 - Lint your commit messages

> [input] reads from stdin if --edit, --env, --from and --to are omitted

> [input] is required: supply via stdin, or --env or --edit or --from and --to
> husky - commit-msg hook exited with code 1 (error)

The only way I could bypass it was to pass in the `--no-verify` flag to my commit message. Still, that defeats the purpose of commitlint and Husky if we're not going to verify our commits. Let me know if you also get a similar error with commitlint whenever this branch gets pushed into main. If so, the new `--no` flag might be the issue.

Reply:
@CasuallyCaffeinated  - Yeah that is interesting. I'd actually be worried to merge this PR if that problem exists as we would need to communicate with the entire team to also use the --no-verify flag to unblock them

If you don't get an error with --no-install, let's roll with that for now? and I'll investigate it after it merges

Reply:
@aetheode Yeah, I tried one more experimental change. Whenever the flag `--no` was present, I was unable to commit. However, when I switched it to `--no-install` it functioned properly. I even tried what @OleDakotaJoe had suggested, namely adding `$1` to the end of the command, but that didn't solve it either. So for now, I'd say we need to stick to the flag `--no-install`.